library/std/src/prelude/v1.rs

```diff
@@ -66,7 +64,7 @@ pub use core::prelude::v1::{derive, global_allocator, test};
 // FIXME: _Unstable_ attribute and internal derive macros are not documented because for them
 // rustdoc generates dead links which fail link checker testing.
 #[stable(feature = "builtin_macro_prelude", since = "1.38.0")]
-#[allow(deprecated, deprecated_in_future)]
+#[allow(deprecated)]
 #[doc(hidden)]
 pub use core::prelude::v1::{bench, test_case, RustcDecodable, RustcEncodable};
```

Comment:
I know that these re-exports of `Rustc…codable` no longer need to be, themselves, `#[doc(hidden)]`, given that now the original defs are. But given `bench` and `test_case`, I was wary of splitting the re-exports yet another time. But I have no strong opinions either.